src/protocol/yaml.rs

```diff
@@ -92,7 +92,7 @@ fn adjust_yaml_output(input: Vec<u8>) -> Vec<u8> {
     result
 }
 
-pub fn write_yaml(mut output_stream: Box<io::Write>, yaml: &Yaml) -> R<()> {
+pub fn write_yaml(output_stream: &mut io::Write, yaml: &Yaml) -> R<()> {
```

Comment:
https://doc.rust-lang.org/edition-guide/rust-2018/trait-system/dyn-trait-for-trait-objects.html

```
Box<Trait> becomes Box<dyn Trait>
&Trait and &mut Trait become &dyn Trait and &mut dyn Trait
```

This looks great! If possible, I might recommend `&mut dyn io::Write` instead :question:

Reply:
Done.